metagpt/skills/CalendarSkill/AssistantShowCalendarEvents/config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
suggest to add a statement in `skill/__init__.py` to ref to the `semantic-kernel/samples/skills`

Reply:
And `skills folder` seems not to be a "from ... import..." part, is it more proper to be provided in `examples/skills`?

Reply:
> suggest to add a statement in `skill/__init__.py` to ref to the `semantic-kernel/samples/skills`

@femto suggest to add a statement in skill/__init__.py to ref to the semantic-kernel/samples/skills.